### PR TITLE
refactor shutter speed formatting code

### PR DESCRIPTION
**Pull request by hatsunearu**
the shutter speed formatting code is repeated in various parts of the code, and in one point incorrectly repeated. 

I put the shutter speed formatting code in variables.c, and it is used everywhere that requires shutter speed formatting.

I also fixed the capital F in the metadata aperture printing.

### CODE REVIEW

File: src/common/variables.h [hunk 1]
```diff
@@ -61,6 +61,8 @@ char *dt_variables_expand(dt_variables_params_t *params, gchar *source, gboolean
 /** reset sequence number */
 void dt_variables_reset_sequence(dt_variables_params_t *params);
 
+char *dt_format_exposure(float exposuretime);
```
Comment (TurboGit) [inline src/common/variables.h:64]: As you see the convention is to use dt_variables_... in variables.h.

I also feel that such common routine should not be declared here. Maybe in utility.h ?

`dt_util_format_exposure(const float exposuretime);`

Or maybe another proposal ?

**Comment by hatsunearu** [comment]
i agree. i will work on putting this in util. i was actually looking for a better location to put it in, haha

**Comment by hatsunearu** [comment]
@TurboGit looks like everything checks out, let me know if there are other issues, i'll be going to sleep lol

**Comment by TurboGit** [comment]
Looks good to me now. Thanks.

**Pull request merged**